src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    // made BOUND = 100 to generate random int which will be more than 0, because we
+    // can't have figure shapes or area lower than 0
+    public static final int BOUND = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Figure defaultFigure = new Circle(Color.WHITE.name(), 10.0);

```

Comment:
let's create this figure in the public method, and exact its properties to DEFAULT_COLOR, DEFAULT_RADIUS constants